packages/openapi-typescript/test/transform/webhooks-object.test.ts

```diff
@@ -105,10 +105,10 @@ describe("transformWebhooksObject", () => {
                 schema: { type: "string" },
                 required: true,
               },
-              { $ref: "#/components/parameters/query/utm_source" },
-              { $ref: "#/components/parameters/query/utm_email" },
-              { $ref: "#/components/parameters/query/utm_campaign" },
-              { $ref: "#/components/parameters/path/version" },
```

Comment:
I feel like these refs were artificially-constructed to match the expected ts output, rather than extracted from a real OpenAPI ref.

Therefore, now that the parameter `in` is respected, I made them more realistic, and adjusted the custom resolver and expected test output.

Reply:
Yup that seems right. I‘ve operated off the assumption that parameters in, say, `query` vs `path` are allowed to share names. But here in `#/components/parameters` you’re right there’s no such thing as “query” or “path”